test/IntegrationFeedTest.php

```diff
@@ -119,14 +120,16 @@ public function testSignedPostHttp()
     public function testReadonlyToken()
     {
         $token = $this->user1->getReadonlyToken();
-        $this->assertSame($token, "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhY3Rpb24iOiJyZWFkIiwiZmVlZF9pZCI6InVzZXIxMSIsInJlc291cmNlIjoiKiJ9.3TVyF2nOiVd_KbOZzJYHabuMxnXy2HFSI--aFAXPMkk");
+        $this->assertSame($token, "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhY3Rpb24iOiJyZWFkIiwiZmVlZF9pZCI6InVzZXIxMSIsInJlc291cmNlIjoiKiJ9.mlBxtn7Ta5muS40HZhJbMHcvwlWuxpKecqz4WQyoErw");
+        //"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhY3Rpb24iOiJyZWFkIiwiZmVlZF9pZCI6InVzZXIxMSIsInJlc291cmNlIjoiKiJ9.3TVyF2nOiVd_KbOZzJYHabuMxnXy2HFSI--aFAXPMkk");
     }
 
     public function testAddActivity()
     {
         $activity_data = ['actor' => 1, 'verb' => 'tweet', 'object' => 1];
         $response = $this->user1->addActivity($activity_data);
         $activity_id = $response['id'];
+        sleep(3);
```

Comment:
Why did you add sleeps instead of removing them? The QA server should not require sleeps anymore.